### PR TITLE
ci(build): split build to speed up build time and enhance step error identification

### PR DESCRIPTION
**Pull request by lpegoraro**
This adds split build and enhances the identification on which step a build error could occur.

### CODE REVIEW

File: ui/src/main.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright Akveo. All Rights Reserved.
+ * Copyright Akveo. All Rights Reserved. test
```
Comment (weyrick) [inline ui/src/main.ts:3]: ?

File: fleet/mocks/agent.go [hunk 2]
```diff
@@ -15,20 +15,20 @@ type agentRepositoryMock struct {
 	agentsMock map[string]fleet.Agent
 }
 
-func (a agentRepositoryMock) SetStaleStatus(ctx context.Context, minutes time.Duration) (int64, error) {
+func (a agentRepositoryMock) SetStaleStatus(_ context.Context, _ time.Duration) (int64, error) {
```
Comment (weyrick) [inline fleet/mocks/agent.go:18]: was this stuff just to fix warnings?

**Review by weyrick — COMMENTED** [review]
This is too big of a change for me to comprehend so I will have to trust that the semantics of the build and publish stay the same. We do still need to be able to build just orb-agent when a new pktvisor build is available though, is that still possible?

File: fleet/mocks/agent.go (hunk 2 above)
Comment (lpegoraro) [reply fleet/mocks/agent.go:18]: yes and also flag the filter of changes on fleet

File: ui/src/main.ts (hunk 1 above)
Comment (lpegoraro) [reply ui/src/main.ts:3]: same, just to trigger the execution on I will remove afterwards

**Comment by lpegoraro** [comment]
> This is too big of a change for me to comprehend so I will have to trust that the semantics of the build and publish stay the same. We do still need to be able to build just orb-agent when a new pktvisor build is available though, is that still possible?

Yeah, basically what we did there was just to split in several jobs. 

The hook of the pktvisor is still in place, this only changes to develop yet. Once this is ok and acknowledged, by all we will move to the other branches

**Pull request closed without merging**